src/why3task.ml

```diff
@@ -0,0 +1,39 @@
+(** Handling tasks in why3. *)
+
+(** [declare_symbols l] add the declaration of every symbol in [l] to [tsk] *)
+let declare_symbols : Why3.Term.lsymbol list -> Why3.Task.task = fun l ->
+    List.fold_left (fun a b -> Why3.Task.add_param_decl a b) None l
```

Comment:
You can eta-reduce to `List.fold_left Why3.Task.add_param_decl a b None l`.

Reply:
Fixed in 8a08c49bbd29eb57e755610901184b9d45e75fae